content/docs/rfcs/22/README.md

```diff
@@ -0,0 +1,52 @@
+---
+slug: 22
+title: 22/TOY-CHAT
+name: Waku v2 Chat
+status: raw
+category: application
+editor: Franck Royer <franck@status.im>
+contributors:
+   - Hanno Cornelius <hanno@status.im>
+---
+
+**Content Topic**: `dingpu`.
+
+This specification explains a toy chat example using Waku v2.
+This protocol is mainly used to:
+
+1. Dogfood Waku v2,
+2. Show an example of how to use Waku v2.
+
+Currently, all main Waku v2 implementations support the toy chat protocol:
+[nim-waku](https://github.com/status-im/nim-waku/blob/master/examples/v2/chat2.nim),
+js-waku ([NodeJS](https://github.com/status-im/js-waku/tree/main/examples/cli-chat) and [web](https://github.com/status-im/js-waku/tree/main/examples/web-chat))
+and [go-waku](https://github.com/status-im/go-waku/tree/master/examples/chat2).
+
+Note that this is completely separate from the protocol the Status app is using for its chat functionality.
+
+# Design
+
+The chat protocol enables sending and receiving messages in a unique chat room.
+The messages SHOULD NOT be encrypted.
+
+The `contentTopic` MUST be set to `dingpu`.
```

Comment:
Is it still `dingpu`? I think it is going to change to `/waku/2/huilong/proto` https://discord.com/channels/624205794384281629/723487577973456896/847057335061839892